docs/Getting Started/Example Apps/index.md

```diff
@@ -12,4 +13,208 @@ The example apps implement soft buttons, template text and images, a main menu a
+
+### Building the Project
+
+For TCP, you may use the built-in Android emulator or an Android phone on the same network as Sdl Core. For Bluetooth, you will need an Android phone that is paired to a TDK or head unit via Bluetooth.
+
+!!! MUST
+Make sure Sdl Core and the HMI are running prior to running Hello Sdl Android
+!!!
+
+Run the project in Android Studio, targeting the device you want Hello Sdl Android installed on.
+
+Hello Sdl Android should compile and launch on your device of choosing:
+
+![Main Screen](assets/mainscreen.png)
+
+Following this, you should see an application appear on the TDK or HMI. In the case of the Generic HMI (using TCP), you will see the following:
+
+![HMI Apps](assets/hmi1.png)
+
+Click on the Hello Sdl icon in the HMI.
+
+![HMI Apps](assets/hmi2.png)
+
+This is the main screen of the Hello Sdl App. If you get to this point, the project is working.
+
+On the device you are running the app on, a lock screen should now appear once the app is opened on the HMI if distracted driver notifications are set to `DD_On`:
+
+![Lockscreen](assets/lockscreen.png)
+
+!!! NOTE
+Lock Screens are an important part of Sdl enabled applications. The goal is to keep the driver's eyes forward and off of the device
+!!!
+
+At this point Hello Sdl Android has been compiled and is running properly! Continue reading through our guides to learn about all of the
+RPCs (Remote Procedure Calls) that can be made with the library.
+
+### Troubleshooting
+
+Sometimes things don't always go as planned, and so this section exists. If your app compiles and does NOT show up on the HMI, there are a few things to check out.
+
+#### TCP
+
+1. Make sure that you have changed the IP in ```SdlService.java``` to match the machine running Sdl Core. Being on the same network is also important.
+2. If you are sure that the IP is correct and it is still not showing up, make sure the Build Flavor that is running is ```tcpDebug```.
+3. If the two above dont work, make sure there is no firewall blocking the incoming port ```12345``` on the machine or VM running SDL Core. In the same breath, make sure your firewall allows that outgoing port.
+4. There are different network configurations needed for different virtualization software (virtualbox, vmware, etc). Make sure yours is set up correctly. Or use [Manticore](https://smartdevicelink.com/resources/manticore/).
+
+#### Bluetooth
+
+1. Make sure the build flavor ```multi_sec_offDebug ``` is selected.
+2. Ensure your phone is properly paired with the TDK
+3. Make sure Bluetooth is turned on - on Both the TDK and your phone
+4. Make sure apps are enabled on the TDK (in settings)
+!@
+
+@![javaEE, javaSE]
+### Introduction
+
+In this guide we take you through the steps to get our sample project, Hello Sdl, running and connected to Sdl Core as well as showing up on the generic HMI.
```

Comment:
The javaSE page for this section is showing javaEE and links to the javaEE project.